### PR TITLE
Search customers by city#80

### PR DESCRIPTION
**Pull request by KaterinPerdom**
Please pull and test locally with behave 
Screenshot 2022-11-29 at 1 12 55 PM

### CODE REVIEW

File: service/static/js/rest_api.js [hunk 1]
```diff
@@ -179,5 +179,84 @@ $(function () {
     // ****************************************
     // Search for a Customer
     // ****************************************
+    $("#search-btn").click(function () {
+
+        let firstname = $("#customer_firstname").val();
```
Comment (mm12063) [inline service/static/js/rest_api.js]: Not needed for this issue. Can be deleted.

File: service/static/js/rest_api.js [hunk 2]
```diff
@@ -179,5 +179,84 @@ $(function () {
...
+        let city = $("#customer_city").val();
+
+        let queryString = ""
+
+        if (firstname) {
```
Comment (mm12063) [inline service/static/js/rest_api.js]: This if statement block is not needed for this issue. Can be deleted.

File: service/static/js/rest_api.js [hunk 3]
```diff
@@ -179,5 +179,84 @@ $(function () {
...
+
+        let ajax = $.ajax({
+            type: "GET",
+            url: `/customers?${queryString}`,
+            contentType: "application/json",
+            data: ''
+        })
+
+        ajax.done(function(res){
+            //alert(res.toSource())
```
Comment (mm12063) [inline service/static/js/rest_api.js]: Please delete this commented out line

**Review by mm12063 — COMMENTED** [review]
I've added some comments. 
The table update code looks better, thank you

**Comment by KaterinPerdom** [comment]
@mm12063 Done with the changes

**Comment by KaterinPerdom** [comment]
@mm12063 
Done with the changes :)

**Pull request merged**